consoles/node-network.html

```diff
@@ -0,0 +1,61 @@
+{{ template "head" . }}
+
+{{ template "prom_right_table_head" }}
+  <th colspan="2">Interface</th>
+</tr>
+{{ range printf "node_network_transmit_bytes{job='node',instance='%s'}" .Params.instance | query | sortByLabel "device" }}
+  <th colspan="2">{{ .Labels.device }}</th>
+  <tr>
+    <td>Transmit Rate</td>
+    <td>{{ template "prom_query_drilldown" (args (printf "rate(node_network_transmit_bytes{job='node',instance='%s',device='%s'}[5m])" .Labels.instance .Labels.device) "B/s" "humanize1024") }}</td>
```

Comment:
I was planning on adding nfs/collectd stats here for my own use case, that was the intent of this page, but it may not be what others want.